File: app/src/main/java/com/projectfox/foxoff/brain/SleepState.kt
package com.projectfox.foxoff.brain

/**
 * Represents the determined sleep state of the user.
 */
enum class SleepState {
    /** User is fully awake (0-40%). */
    AWAKE,
    /** Light fatigue detected (40-70%). */
    DROWSY,
    /** Pre-sleep state (70-90%). */
    PRE_SLEEP,
    /** Confirmed or highly probable sleep (90%+). */
    ASLEEP
}

File: ios/FoxCoreKit/Sources/FoxCoreKit/SleepState.swift
/// Portage direct de `com.projectfox.foxoff.brain.SleepState` (Kotlin).
public enum SleepState {
    /// Utilisateur pleinement éveillé (0-40%).
    case awake
    /// Fatigue légère détectée (40-70%).
    case drowsy
    /// État de pré-sommeil (70-90%).
    case preSleep
    /// Sommeil confirmé ou hautement probable (90%+).
    case asleep
}
